index.html

```diff
@@ -0,0 +1,154 @@
+<html>
+    <head>
+        <title>Microverse</title>
+        <link rel="stylesheet" href="styles.css"/>
+      </head>
+<body>
```

Comment:
- [ ]  [REQUIRED] You should use the HTML5 structure so you are updated and don't have any issues in the future.